manifests/source.pp

```diff
@@ -18,7 +18,7 @@
   $key_content       = undef,
   $key_source        = undef,
   $trusted_source    = undef,
-  $notify_update     = undef,
+  $notify_update     = true,
```

Comment:
@hunner: I don't see where this issue is related to the problem. This is not a `create_resources` problem and I saw it with "3.7.2" and future parser. Would you mind to elaborate?